blueprints/networking/f5-lb/main.tf

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  # F5 configs
+  as3_url  = "https://github.com/F5Networks/f5-appsvcs-extension/releases/download/v3.46.0/f5-appsvcs-3.46.0-5.noarch.rpm"
```

Comment:
this is super ugly, can we find a way to make it not hurt readers' eyes? I can give it a shot if you want; probably moving these in the actual startup script, then tokenizing what you pass in and doing the final manipulations in the template vars?

Reply:
I know... this is the only way I have to cleanly use the original script from F5 and possibly update it over time.
If I start hardcoding these in the script I'll have to redo the whole work every time I update it. Can we keep things here as they are?

Reply:
think that in the original F5 scripts these values were directly injected in the VMs metadata :(

Reply:
I see your point. I still think this might be beautified a bit, if you don't mind I can give it a try once everything else is ready.

Reply:
Sure thing!